tests/link/numba/test_basic.py

```diff
@@ -233,7 +233,7 @@ def assert_fn(x, y):
     numba_res = aesara_numba_fn(*inputs)
 
     # Get some coverage
-    eval_python_only(fn_inputs, fn_outputs, inputs, mode=numba_mode)
+    # eval_python_only(fn_inputs, fn_outputs, inputs, mode=numba_mode)
```

Comment:
This fails for reasons I haven't explored. It seems like it may need special logic for the multi-output Elemwises?